lib/annotate_rb/model_annotator/annotation/annotation_builder.rb

```diff
@@ -46,9 +47,34 @@ def build
 
           private
 
+          def enum_annotations
+            return [] unless @model.has_enums?
+
+            [
+              Components::Base.new.tap do |c|
+                def c.to_default
```

Comment:
Could the methods be extracted to a separate class as to not use metaprogramming here?

Same as lines 61-62